### DISCUSSION TITLE
Introduce `em` to `px` helper to work with breakpoints

### DISCUSSION THREAD

**Discussion by mcmxcdev** [mantinedev/mantine#3845] [Feature requests]
We have specific code for e.g. our sidebar which toggles based on the current screen resolution. We use `MediaQuery` where possible but sometimes can't avoid using code like this:

With mantine v5
```
const { width } = useViewportSize();
const theme = useMantineTheme();

const isLessThanMd = width < theme.breakpoints.md;
```

With mantine v6
```
const { width } = useViewportSize();
const theme = useMantineTheme();

const isLessThanMd =
  Number(em(width).split('em')[0]) <
  Number(theme.breakpoints.md.split('em')[0]);
```

***

As you can see, with v6 `theme.breakpoints.md` is in em unit while `width` from `useViewportSize` is in `px`. There is no conversion helper from `em` to `px` on https://mantine.dev/styles/rem/ that we could use, so we have to do some hacky calculations ourselves here

We need to compare `px` (number) and not `em` or `rem` (string) to make sure the bigger/lesser than calculation is correct.

**Comment by rtivital** [comment]
You can use `getBreakpointValue` function exported from `@mantine/core` package

```ts
import { getBreakpointValue } from '@mantine/core';

getBreakpointValue('10em'); // -> 160px
```

**Reply by mcmxcdev** [reply]
Ah perfect thanks!

When searching through the documentation, there are zero resutls for `getBreakpointValue`. So it might make sense to put it somewhere!
